setup.py

```diff
@@ -14,7 +14,7 @@
   author=__author__,
   author_email='oss@keygen.sh',
   license='MIT',
-  install_requires=['winregistry; sys_platform == "win32"'],
+  install_requires=['winregistry>=2.01; sys_platform == "win32"'],
```

Comment:
Can we add a < v3 constraint here too? So we can avoid this issue again in the future.

Reply:
Hi, of course I'll do it